common/src/main/java/com/wynntils/features/utilities/PerCharacterGuildContributionFeature.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.utilities;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.Persisted;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.Config;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.core.persisted.config.HiddenConfig;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.handlers.chat.event.ChatMessageReceivedEvent;
+import com.wynntils.models.character.event.CharacterUpdateEvent;
+import com.wynntils.utils.mc.McUtils;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.UTILITIES)
```

Comment:
I'm a bit behind in the reviewing here, but should this really be enabled by default? Now, I have not really been playing with a guild, so maybe this makes all sense and everyone assumes that this is how build contributions really should have worked from Wynncraft's side. But this will radically change how `/guild xp` work, and you will get no feedback whatsoever that this is the case.

I usually don't advocate having features turn off by default, but then again, most features are very noticeable. 

A compromise, or perhaps better solution, could be to follow up the "You will now contribute..." message with another message from Wynntils, saying like "This will only apply to the current character. <onClick runCommand=`/feature disable PerCharacterGuildContribution`>Click here</onClick> to disable this functionality", whenever the change was not triggered by us so commandSentFromHere  == true.

Reply:
I did think about having it start as disabled but decided against it as imo it should work like this, as going from a 106 character to a brand new one you have to remember to run the command to contribute less, and vice versa.

I think the best compromise would be to not redirect the message and instead cancel it and replace it with what you suggested, but still keep the config to not display when the feature runs the command.

Reply:
![Screenshot 2023-08-24 132426](https://github.com/Wynntils/Artemis/assets/8337467/fc61f023-7ddf-4d5b-b6f6-628eb239bc5b)

How does this look?

Reply:
I think it looks good.